IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/SSNewsCollectionCellVM.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SSNewsCollectionCellVM.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+final class SSNewsCollectionCellVM {
```

Comment:
Yes, I know that the singular is: a piece of news. But the name will be very long. I will use one of the options you've mentioned.